app/serializers/item_serializer.rb

```diff
@@ -2,12 +2,10 @@ class ItemSerializer < ActiveModel::Serializer
   attributes :id, :name, :date_created, :date_modified, :done
 
   def date_created
-    # Date.parse(object.created_at).strftime("%Y-%m-%d %l:%M %P")
-    object.created_at
+    DateTime.parse(object.created_at.to_s).strftime("%Y-%m-%d %l:%M %P")
```

Comment:
Do not use `DateTime.parse.strftime` without zone. Use one of `Time.zone.parse.strftime`, `DateTime.parse.strftime.current`, `DateTime.parse.strftime.in_time_zone`, `DateTime.parse.strftime.utc`, `DateTime.parse.strftime.getlocal`, `DateTime.parse.strftime.iso8601`, `DateTime.parse.strftime.jisx0301`, `DateTime.parse.strftime.rfc3339`, `DateTime.parse.strftime.to_i`, `DateTime.parse.strftime.to_f` instead.